sdk/communication/Azure.Communication.CallingServer/src/Models/Events/CallingServerEventBase.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary>
+     /// The CallingServer Event Base.
+     /// </summary>
+    public partial class CallingServerEventBase
```

Comment:
All Events seems to have 1 common property - AcsEventType - is it possible to include that in here in base class?
the benefit is that the SDK user can easily use base classe's EventType to figure out which event it parsed into

Reply:
Added AcsEventType to the EventBase class. I had the field non-nullable instead of using `AcsEventType?` do u foresee any problem with that? My reasoning is that we should always construct an Event with an AcsEventType. Do you see this being too restrictive?

Reply:
This is good - one of ACS review board mentioned, considering making this non-nullable. Thank you for pointing out.